generators/server/templates/src/main/resources/application-prod.yml.ejs

```diff
@@ -129,7 +129,7 @@ thymeleaf:
 # ===================================================================
 micronaut:
   server:
-    port: <%= serverPort %>
```

Comment:
Mostly not relevant for the ci pipeline, but we should keep `<%= serverPort %>` it is supposed to come from the main generator